modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -1904,8 +1904,8 @@
                     "id": "1",
```

Comment:
Please remove this file: production swagger.

Reply:
Removed.